aws/rds/variables.tf

```diff
@@ -82,6 +82,11 @@ variable "port" {
   default = ""
 }
 
+variable "publicly_accessible" {
+  description = "Bool to control if instance is publicly accessible."
+  default = true
```

Comment:
I would rather make the default value `false` just to ensure the whoever uses the module really wants it public. This would probably require some more changes on https://github.com/tablexi/chef-repo/.

What do you think about that?

Reply:
Can I do that in a separate PR, so that this PR only focuses on unblocking https://github.com/tablexi/chef-repo/pull/1990?

Instead of updating every user in chef-repo to override the default, I’d actually like to update the RDS instances, so we don’t have surprises like the one I saw this past weekend. That way, we won’t have a default in this repo that’s overridden by *every* use of this module except one 😄 What do you think?

Reply:
I think it is a valid approach also 👍